### PR TITLE
Upgrade to libvips v8.12.0-rc1

### PR DESCRIPTION
**Pull request by lovell**

### CODE REVIEW

File: THIRD-PARTY-NOTICES.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ used under the terms of the following licences:
 |---------------|-----------------------------------------------------------------------------------------------------------|
 | aom           | BSD 2-Clause + [Alliance for Open Media Patent License 1.0](https://aomedia.org/license/patent-license/)  |
 | cairo         | Mozilla Public License 2.0                                                                                |
+| cgif          | MIT Licence                                                                                               |
 | expat         | MIT Licence                                                                                               |
 | fontconfig    | [fontconfig Licence](https://gitlab.freedesktop.org/fontconfig/fontconfig/blob/main/COPYING) (BSD-like) |
```
Comment (kleisauke) [inline THIRD-PARTY-NOTICES.md:12]: nit: missing spaces
```suggestion
| fontconfig    | [fontconfig Licence](https://gitlab.freedesktop.org/fontconfig/fontconfig/blob/main/COPYING) (BSD-like)   |
```

File: build/lin.sh [hunk 2]
```diff
@@ -468,6 +470,13 @@ fi
   ${DARWIN:+--disable-Bsymbolic}
 make install-strip
 
+mkdir ${DEPS}/cgif
+$CURL https://github.com/dloebl/cgif/archive/V${VERSION_CGIF}.tar.gz | tar xzC ${DEPS}/cgif --strip-components=1
+cd ${DEPS}/cgif
+CFLAGS="${CFLAGS} -O3" meson setup _build --default-library=static --buildtype=release --strip --prefix=${TARGET} ${MESON}
```
Comment (kleisauke) [inline build/lin.sh:476]: Maybe we should avoid compiling the unit tests?
```suggestion
CFLAGS="${CFLAGS} -O3" meson setup _build --default-library=static --buildtype=release --strip --prefix=${TARGET} ${MESON} \
  -Dtests=false
```

File: build/lin.sh [hunk 3]
```diff
@@ -124,6 +124,7 @@ VERSION_PANGO=1.49.3
 VERSION_SVG=2.52.4
 VERSION_AOM=3.2.0
 VERSION_HEIF=1.12.0
+VERSION_CGIF=0.0.2
```
Comment (kleisauke) [inline build/lin.sh:127]: This is currently missing in `versions.json`.

**Review by kleisauke — APPROVED** [review]
LGTM % comments

File: build/lin.sh (hunk 3 above)
Comment (lovell) [reply build/lin.sh:127]: Added, thanks.

**Pull request merged**